### ISSUE TITLE
[Help] Nsbin.bat and several questions

### ISSUE THREAD

**Issue by Cloudhax23** [3F/DllExport#39]
After adding DllExport from nuget I get this error. 
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		The command "NSBin.bat "C:\Users\Cloud\Documents\Visual Studio 2017\Projects\DllExportTest\packages\DllExport.1.5.2\tools\..\lib\net20\DllExport.dll" "System.Runtime.InteropServices" "true"" exited with code 1.	DllExportTest
```
Also my other question is why is the namespace so long or have i done something incorrectly? https://gyazo.com/a1650335764e5e4b3f0b447c0bdd2b57

Next, are there any example projects of using DllExport? I attempted to bypass the NSBIN.bat by simply clearing the code in the batch file out. However once compiled i used IDA and DependencyWalker to see if the export was actually exported and did not see anything.

And lastly, i see in some of your screenshots there is a configurator. I, however, do not see this in VS2017 so is there a fix for that?

**Comment by 3F** [comment]
Please read here #33
Currently I'll consider new way of distribution, why is so here  - #38

I already started this - https://github.com/3F/MvsSln
the new license and common version will be soon, to continue #38 (comment new way there if needed)

**temporarily** try to configure it manually as it described in #33

**Comment by jpeg729** [comment]
I had the same problem until I upgraded PowerShell.

NSBin.bat was failing to load NSBin.dll because it "was compiled with a newer runtime".

**Comment by Cloudhax23** [comment]
This is correct. Upgrading the PS allowed me to get the gui and everything compiled fine. Thanks!

**Comment by 3F** [comment]
@Cloudhax23 Did I understand correctly that you had problem - "a runtime newer than the currently loaded" (#41) for this error ? or this another problem, but was resolved by the same solution ?

**Comment by jpeg729** [comment]
To clarify NSBin.bat tries to load NSBin, but PowerShell complains that NSBin was `built by a runtime newer than the currently loaded` and that is why NSBin.bat `exited with code 1.`

The error is the same, but the error messages shown in different places have different levels of detail.

The fix is upgrading PowerShell.

I don't know if it is necessary, but I did also install msbuild tools. [https://www.microsoft.com/en-us/download/details.aspx?id=48159](https://www.microsoft.com/en-us/download/details.aspx?id=48159)

**Comment by 3F** [comment]
@jpeg729 thanks for the additional information

**Comment by 3F** [comment]
I closed this issue because of new [Wizard for v1.6+ versions](https://github.com/3F/DllExport/issues/38). You already can test it via our beta releases:

* Latest manager: https://3F.github.io/DllExport/releases/latest/manager/
* Please note again - you shouldn't use standard nuget clients anymore: https://www.youtube.com/watch?v=okPThdWDZMM 

If you still have a some related problems, please create new issue for new planned Wizard.
